lib/recursive_open_struct.rb

```diff
@@ -155,6 +168,12 @@ def delete_field(name)
 
   private
 
+  unless OpenStruct.public_instance_methods.include?(:initialize_copy)
```

Comment:
Any particular reason this is down here, vs being an `else` to the check further up around `initialize_copy`?

Reply:
As far as detecting goes, `OpenStruct` now has a version constant that could be used. I should update it to a prerelease version to facilitate this.

Reply:
Honestly I tend to prefer feature checking than version testing these days, especially with the various alternative ruby implementations.

> Any particular reason this is down here

To make it a private method, rather than a public one.